amp/repository/aim/view/viewActivityDifferences.jsp

```diff
@@ -141,7 +141,69 @@
 						</logic:iterate>
 
 					</td></tr>
-				</logic:iterate>	
+				</logic:iterate>

```

Comment:
same as wessi we should reuse as an include the jsp part that iterates over a single activity so we don't have duplicate code. Please refactor it so we include the duplicate code.and this way its easier to maintain